app/src/ui/layout/container.js

```diff
@@ -34,7 +36,8 @@ export default function Container({
       className={cx('container', {
         [`align-${align}`]: !!align,
         [`justify-${justify}`]: !!justify,
-        row: !!row
+        row: !!row,
+        'scroll-y': !!scrollable
```

Comment:
More nit-picky-ness, could this be above `row` next to the other longer pairs? 🙂